src/serializer/ResidualFunctions.js

```diff
@@ -249,49 +255,69 @@ export class ResidualFunctions {
           invariant(t.isCallExpression(funcNode)); // .bind call
           body = getFunctionBody(instance);
         }
+        let pushToBody = elem => {
+          if (instance.declarationBodyOverride) instance.declarationBodyOverride.unshift(elem);
```

Comment:
I am confused that the code above obtains a `body`, but if `instance.declarationBodyOverride` that body is ignored here?